roverParams_maxAngAccel: 0.7
roverParams_maxLinAccel: 1.0
roverParams_maxLinDecel: 1.4
roverParams_maxV: 1.0
roverParams_minV: 0.0
roverParams_maxW: 1.0
roverParams_robotLength: 1.07
roverParams_robotWidth: 1.03
roverParams_timestep: 0.5
local_planner/costmap_topic: /costmap
local_planner/goal_gps_topic: /goal/utm
local_planner/odometry_topic: /odometry/rover_gps_odom
local_planner/pose_topic: /localization/pose_utm

local_planner/velocity_out_topic: /cmd_vel #/local_planner/cmd_vel
local_planner/status_topic: /local_planner/status
local_planner/enable_topic: /local_planner/enable
local_planner/goal_reached_distance: 2.0
local_planner/goal_in_range_distance: 10.0
local_planner/height_thresh: 100
local_planner/dwa/heading_weight: 1.7
local_planner/dwa/distance_weight: 0.8
local_planner/dwa/velocity_weight: 0.2

